{"uid":"fa5a5fd142ade7ea","name":"test_setup","fullName":"tests.integration.test_smoke#test_setup","historyId":"d623373ae070a62cab8e4b2f9dd522ae","time":{"start":1751505842363,"stop":1751505843494,"duration":1131},"status":"broken","statusMessage":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 31, in test_setup\n    await ops_test.model.deploy(\"./releases/latest/postgresql-k8s-bundle.yaml\", trust=True)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 1914, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 453, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 436, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 2078, in _resolve_charm\n    resp = await charms_facade.ResolveCharms(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 482, in wrapper\n    reply = await f(*args, **kwargs)\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/_client6.py\", line 276, in ResolveCharms\n    reply = await self.rpc(msg)\n            ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 680, in rpc\n    result = await self.connection.rpc(msg, encoder=TypeEncoder)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/connection.py\", line 597, in rpc\n    raise errors.JujuAPIError(result)\njuju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1751505842125,"stop":1751505842125,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1751505842125,"stop":1751505842362,"duration":237},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1751505842125,"stop":1751505842125,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1751505842363,"stop":1751505842363,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 31, in test_setup\n    await ops_test.model.deploy(\"./releases/latest/postgresql-k8s-bundle.yaml\", trust=True)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 1914, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 453, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 436, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 2078, in _resolve_charm\n    resp = await charms_facade.ResolveCharms(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 482, in wrapper\n    reply = await f(*args, **kwargs)\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/_client6.py\", line 276, in ResolveCharms\n    reply = await self.rpc(msg)\n            ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 680, in rpc\n    result = await self.connection.rpc(msg, encoder=TypeEncoder)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/connection.py\", line 597, in rpc\n    raise errors.JujuAPIError(result)\njuju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","steps":[],"attachments":[{"uid":"5777e2cfbc694585","name":"log","source":"5777e2cfbc694585.txt","type":"text/plain","size":1095}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1751505843526,"stop":1751505843847,"duration":321},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1751505843526,"stop":1751505843526,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1751505843847,"stop":1751505843847,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1751505843847,"stop":1751505843847,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1751505843847,"stop":1751505843848,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"fv-az1690-341"},{"name":"thread","value":"16353-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"4c8d7f276b45d6d0","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":41,"skipped":0,"passed":319,"unknown":9,"total":369},"items":[{"uid":"75fb2a4d0c82bcec","reportUrl":"../2148//#testresult/75fb2a4d0c82bcec","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751419436352,"stop":1751419437078,"duration":726}},{"uid":"d7c1cb3b49ec972a","reportUrl":"../2146//#testresult/d7c1cb3b49ec972a","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751333057688,"stop":1751333060620,"duration":2932}},{"uid":"49e58354ae2d06f2","reportUrl":"../2145//#testresult/49e58354ae2d06f2","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751246702440,"stop":1751246706495,"duration":4055}},{"uid":"e909f1a17f53f05e","reportUrl":"../2144//#testresult/e909f1a17f53f05e","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751160241859,"stop":1751160243729,"duration":1870}},{"uid":"261b1141275ef46f","reportUrl":"../2143//#testresult/261b1141275ef46f","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751073811350,"stop":1751073812258,"duration":908}},{"uid":"8c0b34bca34adb4b","reportUrl":"../2142//#testresult/8c0b34bca34adb4b","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750987433053,"stop":1750987435340,"duration":2287}},{"uid":"aff290b2f6983b80","reportUrl":"../2141//#testresult/aff290b2f6983b80","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750901068453,"stop":1750901069982,"duration":1529}},{"uid":"fea63496ff21bcf7","reportUrl":"../2140//#testresult/fea63496ff21bcf7","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750814635320,"stop":1750814637893,"duration":2573}},{"uid":"4d3924f96e2c5d0c","reportUrl":"../2138//#testresult/4d3924f96e2c5d0c","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750728233801,"stop":1750728234253,"duration":452}},{"uid":"bdf8633b6b27bc63","reportUrl":"../2137//#testresult/bdf8633b6b27bc63","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750641871206,"stop":1750641872856,"duration":1650}},{"uid":"8e7ebc0e2b56bc8f","reportUrl":"../2136//#testresult/8e7ebc0e2b56bc8f","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750555449492,"stop":1750555450024,"duration":532}},{"uid":"530c132132d71cb2","reportUrl":"../2135//#testresult/530c132132d71cb2","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750469056951,"stop":1750469061649,"duration":4698}},{"uid":"129b0999e298a272","reportUrl":"../2134//#testresult/129b0999e298a272","status":"passed","time":{"start":1750382670407,"stop":1750383254437,"duration":584030}},{"uid":"b72bfb0c2059ead8","reportUrl":"../2133//#testresult/b72bfb0c2059ead8","status":"passed","time":{"start":1750296241156,"stop":1750296813314,"duration":572158}},{"uid":"6f2b8a8ed946ee46","reportUrl":"../2132//#testresult/6f2b8a8ed946ee46","status":"passed","time":{"start":1750209851354,"stop":1750210436584,"duration":585230}},{"uid":"e0b56891da3d5bf9","reportUrl":"../2129//#testresult/e0b56891da3d5bf9","status":"passed","time":{"start":1750123520920,"stop":1750124143127,"duration":622207}},{"uid":"61195258665448e6","reportUrl":"../2128//#testresult/61195258665448e6","status":"passed","time":{"start":1750037038110,"stop":1750037660753,"duration":622643}},{"uid":"9d124cd8c479c360","reportUrl":"../2127//#testresult/9d124cd8c479c360","status":"passed","time":{"start":1749950683210,"stop":1749951305678,"duration":622468}},{"uid":"9f3ddd83fdf343b5","reportUrl":"../2126//#testresult/9f3ddd83fdf343b5","status":"passed","time":{"start":1749864344595,"stop":1749864961447,"duration":616852}},{"uid":"efbc3172f8f3441","reportUrl":"../2125//#testresult/efbc3172f8f3441","status":"passed","time":{"start":1749777823526,"stop":1749778450316,"duration":626790}}]},"tags":["asyncio","abort_on_fail"]},"source":"fa5a5fd142ade7ea.json","parameterValues":[]}